content/post/proteus.md

```diff
@@ -0,0 +1,115 @@
+---
+author: mvader
+date: 2016-12-22
```

Comment:
Yes, we don't know yet when we are going to publish it so I'll change it when we do.